go/private/providers.bzl

```diff
@@ -15,4 +15,28 @@
 GoLibrary = provider()
 GoBinary = provider()
 CgoLibrary = provider()
-GoPath = provider()
\ No newline at end of file
+GoPath = provider()
+
+def library_attr(mode):
+  """Returns the attribute name for the library of the given mode"""
```

Comment:
Would be good to mention `RACE_MODE` as an example. I don't think it's obvious from the context.

Reply:
I improved the comment(s)